app/views/gems/gem.erb

```diff
@@ -1,5 +1,9 @@
+<% VERSION_LIMIT = 3 %>
```

Comment:
defining a constant in the view does not work, it is evaulated each time the view is rendered